docker/Dockerfile.appsre

```diff
@@ -0,0 +1,49 @@
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5-240 as builder
+
+RUN microdnf install wget tar java-11-openjdk-devel
+
+RUN wget https://archive.apache.org/dist/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.tar.gz -P /tmp \
+    && tar xf /tmp/apache-maven-3.8.1-bin.tar.gz -C /opt \
+    && ln -s /opt/apache-maven-3.8.1 /opt/maven \
+    && mkdir -p /workspace
+
+COPY . /workspace
+
+WORKDIR  /workspace
+
+RUN export JAVA_HOME=/usr/lib/jvm/java-11-openjdk \
+    && export M2_HOME=/opt/maven \
+    && export MAVEN_HOME=/opt/maven \
+    && export PATH=$M2_HOME/bin:$PATH \
+    && sh app-interface/buid_project.sh
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5-240 as runtime
+
+ARG JAVA_PACKAGE=java-11-openjdk-headless
+ARG RUN_JAVA_VERSION=1.3.8
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
+
+RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} \
+    && microdnf update \
+    && microdnf clean all \
+    && mkdir /deployments \
+    && chown 1001 /deployments \
+    && chmod "g+rwX" /deployments \
+    && chown 1001:root /deployments \
+    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
+    && chown 1001 /deployments/run-java.sh \
+    && chmod 540 /deployments/run-java.sh \
+    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
+
+ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/lib/ /deployments/lib/
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/*.jar /deployments/
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/app/ /deployments/app/
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/quarkus/ /deployments/quarkus/
+
+EXPOSE 8080
+USER 1001
+
+ENTRYPOINT [ "/deployments/run-java.sh" ]
```

Comment:
Can you please refactor starting from this https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/docker/Dockerfile.jvm#L78-L93 
(of course making sure to preserve the `--from=builder` etc)?

It seems to be the "latest" version of Quarkus Dockerfile